### DISCUSSION TITLE
这个插件时灵时不灵

### DISCUSSION THREAD

**Discussion by vfjy12** [guopenghui/obsidian-language-learner#90] [General]
大佬我的ymal区写的是对的，但是有些web就是不能成功进入阅读模式，即便是有那个阅读模式的按钮存在也不行
![uTools_1674033126150](https://user-images.githubusercontent.com/121844562/213130650-0dc2abdb-a044-4354-887b-fd1edd301ca4.png)
![uTools_1674033109683](https://user-images.githubusercontent.com/121844562/213130896-ebdeefc4-7941-49b3-beb7-93227c44e975.png)
而且，我加过^^^article和^^^word和^^^note了

**Comment by guopenghui** [comment]
按 `Ctrl + Shift + i` 打开控制台看看有没有什么报错信息吧。一片空白说明解析过程出错了

**Comment by vfjy12** [comment]
大佬，
没有看懂wj量可以确定的是不是空白

guopenghui ***@***.***> 于2023年1月18日周三 21:16写道：

**Reply by naanwoo** [reply]
我也遇到了类似情况，我的case是文章中出现了“---”的分行导致解析错误，FYI
